docs/versioned_docs/version-v3/providers/strava.md

```diff
@@ -24,6 +24,11 @@ providers: [
   Providers.Strava({
     clientId: process.env.STRAVA_CLIENT_ID,
     clientSecret: process.env.STRAVA_CLIENT_SECRET,
+    authorization: {
+        params: {
+          scope: process.env.STRAVA_SCOPES,
+        },
+      },
```

Comment:
```suggestion
    authorization: {
      params: {
        scope: process.env.STRAVA_SCOPES,
      },
    },
```
Within this codebase, we use [`prettier`](https://prettier.io/) for formatting 💅🏽